cfy_manager/components/mgmtworker/cluster/scripts/configure_syncthing_script.py

```diff
@@ -40,8 +47,9 @@ def run_syncthing_configuration(hostname, active_manager_ip,
     args = parser.parse_args()
     with open(args.args_dict_config_path, 'r') as f:
         args_dict = json.load(f)
+
+    config.instance.load_configuration()
+    setup_flask_app(manager_ip=config.instance.postgresql_host)
```

Comment:
manager_ip is postgres IP?

Reply:
Unfortunate naming in setup_flask_app, yes. This is also what the other script has to do (the create/ populate db one)